pkg/cmd/openshift.go

```diff
@@ -30,6 +34,7 @@ func openshiftCmd(vals *types.CmdValues) *cobra.Command {
 				fmt.Fprintln(os.Stderr, err.Error())
 				os.Exit(1)
```

Comment:
@kadel, upon returning err, cobra prints out the help alongside the error too, which will make the output more convoluted. @surajssd and me had this discussion today morning.
How do you suggest we go about this?